src/core/integrators/stokesian_dynamics_inline.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2010-2019 The ESPResSo project
+ *
+ * This file is part of ESPResSo.
+ *
+ * ESPResSo is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * ESPResSo is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "ParticleRange.hpp"
+#include "config.hpp"
+#include "particle_data.hpp"
+#include "rotation.hpp"
+#include "stokesian_dynamics/sd_interface.hpp"
+
+#include "utils/Vector.hpp"
+
+#ifdef STOKESIAN_DYNAMICS
+
+inline void
+stokesian_dynamics_propagate_vel_pos(const ParticleRange &particles) {
+  auto const skin2 = Utils::sqr(0.5 * skin);
+
+  // Compute new (translational and rotational) velocities
+  propagate_vel_pos_sd(particles);
+
+  for (auto &p : particles) {
+
+    // Translate particle
+    p.r.p[0] += p.m.v[0] * time_step;
+    p.r.p[1] += p.m.v[1] * time_step;
+    p.r.p[2] += p.m.v[2] * time_step;
+
+#ifdef ROTATION
```

Comment:
Please remove all `#ifdef ROTATION`. STOKESIAN_DYNAMICS and STOKESIAN_DYNAMICS_GPU imply ROTATION.